docs/releases.md

```diff
@@ -116,7 +126,6 @@ For more details check the documentation page: [mjpg-streamer](../software/camer
 
 - DietPi-Config  :octicons-arrow-right-16: Enabling WiFi + Ethernet adapters, both on different subnets, breaks WiFi connection in some cases: <https://github.com/MichaIng/DietPi/issues/2103>
 - DietPi-Software | [Node-RED](../software/hardware_projects/#node-red)  :octicons-arrow-right-16:  Pre-installed modules cannot be updated via web UI: <https://github.com/MichaIng/DietPi/issues/2073>
-- DietPi-Software | Raspimjpeg  :octicons-arrow-right-16:  With Lighttpd, streaming mjpeg does not work: <https://github.com/MichaIng/DietPi/issues/1747>
```

Comment:
Ah good find. The above Node-RED issue has been solved as well with the change you added the log for above 🙂.

Reply:
Agree on _Raspimjpeg_ . Not sure what you mean for _Node-RED_

Reply:
> Not sure what you mean for Node-RED

The new changelog entry above at line 101 solves the known Node-RED issue at line 128. So that needs to be removed as well.